style.css

```diff
@@ -0,0 +1,558 @@
+	justify-content: flex-start;
+	width: 100%;
+	height: 253px;
+	margin-bottom: 68px;
+}
+
+.top-content__block {
+	display: flex;
+	flex-direction: column;
+	width: 209px;
+	padding-left: 54px;
+	justify-content: center;
+	align-items: flex-start;
+}
+
+.top-content__h1 {
+	font-size: 30px;
+	font-weight: bold;
+	line-height: 29px;
+	text-transform: uppercase;
+	color: #ffffff;
+	margin-bottom: 14px;
+	text-align: left;
+}
+
+.link__white {
+	color: #ffffff;
+	text-align: left;
+}
+
+/*catalog*/
+
+/*.catalog {
+	margin: 0 auto;
+}*/
+
+.catalog__title {
+	font-size: 22px;
+	font-weight: bold;
+	line-height: 21px;
+	text-align: center;
+	text-transform: uppercase;
+	margin-bottom: 15px;
+	font-weight: bold;
+}
+
+.catalog__items {
+	display: flex;
+	justify-content: space-around;
+}
+
+.catalog__item {
+	display: block;
+	position: relative;
+	width: 208px;
+	height: 261px;
+}
+
+.catalog__item:not(:last-child):after {
+	content: "";
+  opacity: .2;
+  background: #000000;
+  height: 101px;
+  position: absolute;
+	top: 36px;
+	right: -15px;
+	width: 1px;
+}
+
+.catalog-item__image {
+	width: 170px;
+	height: 173px;
+	margin-bottom: 24px;
+	padding-left: 21px;
+}
+
+.catalog-item__title {
+	font-size: 12px;
+	line-height: 11px;
+	text-align: left;
+	margin-bottom: 10px;
+	padding-left: 11px;
+}
+
+.catalog-item__description {
+	display: grid;
+	grid-template-columns: 2fr 1fr;
+	padding-left: 11px;
+}
+
+.catalog-item__price {
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	font-size: 14px;
+	font-weight: bold;
+	line-height: 13px;
+	text-align: left;
+}
+
+.catalog-item__varities {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+}
+
+.catalog-item__grey {
+	height: 7px;
+	width: 7px;
+	border-radius: 50%;
+	background-color: #C4C4C4;
+}
+
+.catalog-item__black {
+	height: 7px;
+	width: 7px;
+	border-radius: 50%;
+	background-color: #0E1211;
+}
+
+.catalog-item__red {
+	height: 7px;
+	width: 7px;
+	border-radius: 50%;
+	background-color: #B62E3C;
+}
+
+.catalog-item__orange {
+	height: 7px;
+	width: 7px;
+	border-radius: 50%;
+	background-color: #FDB431;
+}
+
+.catalog-item__brown {
+	height: 7px;
+	width: 7px;
+	border-radius: 50%;
+	background-color: #E29D69;
+}
+
+.catalog-item__cherry {
+	height: 7px;
+	width: 7px;
+	border-radius: 50%;
+	background-color: #762E31;
+}
+
+.catalog-item__blue {
+	height: 7px;
+	width: 7px;
+	border-radius: 50%;
+	background-color: #257DA9;
+}
+
+.active-item {
+	height: 9px;
+	width: 9px;
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+	.wrapper {
+		width: 768px;
+	}
+	.header {
+		grid-template-columns: 1fr 1fr;
+		position: relative;
+	}
+	.header__menu, .header__search, .header__user-block, .header__cart, .header__language-block {
+		display: none;
+	}
+	.header__logo-image {
+		display: none;
+	}
+	.header__logo-image_adaptive {
+		width: 156px;
+		height: 34px;
+		display: flex;
+		align-items: center;
+		margin-left: 43px;
+	}
+	.menu-bar {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)